src/__tests__/patients/view/ViewPatient.test.tsx

```diff
@@ -88,8 +86,6 @@ describe('ViewPatient', () => {
     await setup()
```

Comment:
I keep getting this warning in this test:

```
     Warning: React has detected a change in the order of Hooks called by ViewPatient. This will lead to bugs and errors if not fixed. For more information, read the Rules of Hooks: https://fb.me/rules-of-hooks
      
         Previous render            Next render
         ------------------------------------------------------
      1. useCallback                useCallback
      2. useContext                 useContext
      3. useContext                 useContext
      4. useContext                 useContext
      5. useContext                 useContext
      6. useContext                 useContext
      7. useContext                 useContext
      8. useContext                 useReducer
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      
          in ViewPatient (created by Context.Consumer)
          in withRouter(ViewPatient) (at ViewPatient.test.tsx:71)
          in Route (at ViewPatient.test.tsx:70)
          in Router (at ViewPatient.test.tsx:69)
          in Provider (created by WrapperComponent)
          in WrapperComponent
```

I think is error is caused by this hook. https://github.com/HospitalRun/hospitalrun-frontend/blob/master/src/page-header/button-toolbar/ButtonBarProvider.tsx#L30. Do we need to find a way to get rid of this warning? Any suggestion on how to fix this?